src/features/channels/components/message-reply-provider/MessageReplyProvider.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Please do not use eslint disable rules. If we need to make an exception, we can add new rules or disable rules globally (when there is a good reason to do it)